### PR TITLE
Updates for current version

### PR DESCRIPTION
**Pull request by Kosmas**

### CODE REVIEW

File: docs/testing/testing.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ The first test case we'll look at is `test/controllers/page_controller_test.exs`
 defmodule HelloPhoenix.PageControllerTest do
   use HelloPhoenix.ConnCase
 
-  test "GET /" do
+  test "GET /", %{conn: conn} do
     conn = get conn(), "/"
```
Comment (Gazler) [inline docs/testing/testing.md]: We should change this to `conn = get conn, "/"`

**Comment by Gazler** [comment]
Thanks! Just a small change then this is good to merge.

**Comment by Kosmas** [comment]
@Gazler thanks.
I believe that you can squash the commits  when you merge.
If not let me know.

**Comment by Gazler** [comment]
Thanks <3

**Pull request merged**